io.github.RobertBeckebans.RBDOOM-3-BFG.yaml

```diff
@@ -0,0 +1,126 @@
+app-id: io.github.RobertBeckebans.RBDOOM-3-BFG
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm18
+command: rbdoom3bfg.sh
+finish-args:
+  - --device=all
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+cleanup:
+  - /bin/7zr
+  - /include
+  - /lib/*.a
+  - /lib/*.la
+  - /lib/pkgconfig
+  - /share/cmake
+  - /share/man
+modules:
+
+  # build DirectXShaderCompiler from Git sources
+  - name: directxshader-compiler-build
+    buildsystem: cmake-ninja
+    builddir: true
+    no-make-install: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DDXC_COVERAGE=Off
+      - -C ../cmake/caches/PredefinedParams.cmake
+    build-options:
+      cxxflags: -fvisibility-inlines-hidden -fvisibility=hidden
+    sources:
+      - type: git
+        url: https://github.com/microsoft/DirectXShaderCompiler
+        tag: v1.8.2405
+    post-install:
+      # Install binaries
+      - install -Dm 755 bin/dxc /app/bin/dxc
+      - install -Dm 755 lib/libdxcompiler.so /app/lib/libdxcompiler.so
+      # Install headers
+      - install -D ../include/dxc/dxcapi.h /app/include/dxc/dxcapi.h
+      - install -D ../include/dxc/dxcerrors.h /app/include/dxc/dxcerrors.h
+      - install -D ../include/dxc/dxcisense.h /app/include/dxc/dxcisense.h
+      - install -D ../include/dxc/WinAdapter.h /app/include/dxc/WinAdapter.h
+
+  # build application from Git sources
+  - name: rbdoom3bfg-build
+    buildsystem: cmake-ninja
+    builddir: true
+    subdir: neo
+    no-make-install: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DFFMPEG=Off
+      - -DBINKDEC=On
+      - -DRETAIL=Off
+      - -DFLATPAK=On
+      - -DUSE_PRECOMPILED_HEADERS=Off
+      - -DUSE_INTRINSICS_SSE=Off
+      - -DUSE_INTRINSICS_NEON=Off
+    build-options:
+      arch:
+        x86_64:
+          config-opts:
+            - -DUSE_INTRINSICS_SSE=On
+        aarch64:
+          config-opts:
+            - -DUSE_INTRINSICS_SSE=Off
```

Comment:
shouldn't this have `-DUSE_INTRINSICS_NEON=On` ?

Reply:
-DUSE_INTRINSICS_NEON=On would enable the BC6 texture compression in the engine and that would require to add the ISPC compiler to the build process next to the DXC SPIR-V compiler. I couldn't get this to work. However it is also not really required when people just use the engine for playing the game. The textures I included in this release are already precompressed.